src/GenSymIO.chpl

```diff
@@ -1,6 +1,4 @@
 module GenSymIO {
```

Comment:
Thinking about this a little more, I think that the `GenSymIO` module name is weird for this file now. The functions provide functionality like creating a pdarray with `arrayMsg()` and converting a pdarray to an ndarray with `tondarrayMsg()`, and I'm not sure if these are necessarily "IO" functions, but I don't know what I would call it instead and maybe there is more to the name than I realize.

Reply:
I want to keep the PR scope somewhat confined, so I'm not going to change it in this go-round; we can always rename it in another iteration.